src/handlers/har/index.ts

```diff
@@ -0,0 +1,68 @@
+import { ipcMain, dialog } from 'electron'
+import { readFile, copyFile } from 'fs/promises'
+import path from 'path'
+
+import { RECORDINGS_PATH } from '@/constants/workspace'
+import { HarWithOptionalResponse } from '@/types/har'
+import { browserWindowFromEvent } from '@/utils/electron'
+import { createFileWithUniqueName } from '@/utils/fileSystem'
+
+import { HarHandler } from './types'
+
+export function initialize() {
+  ipcMain.handle(
+    HarHandler.SaveFile,
```

Comment:
No changes were made to the implementation of the handlers other than:

- moving them out of `main.ts`
- replacing the harcoded channel name (e.g. `har:save`) with an enum (e.g. `HarHandler.SaveFile`)